src/utilities/common.ts

```diff
@@ -127,6 +127,10 @@ export const formatApy = (apy?: BigNumber | string | number): string => {
   return `${apyBN.toExponential(2, 1)}%`;
 };
 
+export const boundCommaNumber = commaNumber.bindWith(',', '.');
+export const format = (bigNumber: BigNumber, dp = 2) =>
```

Comment:
Sorry I just caught this now, but I'd suggest renaming this to something more explicit like `formatToCommaNumber`, as otherwise I feel it isn't clear what this actually does